Program.cs

```diff
@@ -1,4 +1,24 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Extension of Spelling Bee assignment
+// Display the first 7 letters to be used as the seed for the Spelling Bee program from the given word list.
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+// Reading all letter in the wordlist.
+string wordsList = File.ReadAllText ("D:/OneDrive - Trumpf Metamation Pvt Ltd/words.txt");

```

Comment:
I asked to use `C:/etc/words.txt` because all of us have the file in that directory. This program will not run in my system unless I have the words.txt in the path you have specified. So, create a folder named `etc` in `C drive` and move the words.txt there.